apollo-example/pom.xml

```diff
@@ -81,6 +81,18 @@
                     </compilerArgs>
                 </configuration>
             </plugin>
+
+            <!-- Do not deploy, this is just an example -->
+            <!-- TODO: Remove or move to one example -->
+            <!-- TODO: set up ci to build and verify this -->
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-deploy-plugin</artifactId>
+              <version>2.8.2</version>
```

Comment:
You shouldn't have to specify the version of the deploy plugin here - it should be defined in some parent pom instead, though maybe it isn't.